src/sonic-build-hooks/scripts/pre_run_buildinfo

```diff
@@ -8,6 +8,8 @@ mkdir -p $BUILD_VERSION_PATH
 mkdir -p $LOG_PATH
 
 [ -d $PRE_VERSION_PATH ] && rm -rf $PRE_VERSION_PATH
+[ -d $POST_VERSION_PATH ] && rm -rf $POST_VERSION_PATH
+mkdir -p $POST_VERSION_PATH
```

Comment:
Why don't mkdir -p PRE_VERSION_PATH?

Reply:
@liushilongbuaa , thanks for your comment, PRE_VERSION_PATH and the POST_VERSION_PATH will be made automatically when collecting the version files, but the POST_VERSION_PATH is made for purging before collecting the versions.